google_amoeba_net_d_quater_tpupod_tensorflow18.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "v1.0",
+  "author": "Google",
+  "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+  "framework": "TensorFlow 1.8.0-rc1",
+  "codeURL": "https://github.com/tensorflow/tpu/tree/e5c126d66aa3d25e0cb066bdf7fc46f98fe59901/models/experimental/amoeba_net/",
+  "model": "AmoebaNet-D N6F256",
+  "hardware": "1/4 of a TPUv2 Pod",
+  "timestamp": "2018-04-20",
+  "misc": {
+    "optimizer": "RMSProp",
+    "movingAverageDecay": 0.9999,
+    "weightDecay": 0.00004,
+    "batchSize": 4096,
+    "warmUp": "the learning rate starts at 0, then it increases linearly per iteration for the first 3 epochs",
+    "exponentialDecay": "The initial learning rate is 10.24, it decrease by a factor of 0.88 every one epoch",
+    "numEpochs": 41,
```

Comment:
Why did the number of epochs increase here? Previously you used 35 epochs for the Cloud TPU and 1/16 of a TPUv2 Pod.

Reply:
It won't train to 93% in 35 epochs. There are more TPU cores in a 1/4 TPUv2 pod but the total batch size is the same. The smaller per-core batch size might affect the batch normalization so it takes more epochs to train 93%.

Reply:
Thanks for clarifying.